### PR TITLE
fix parameter verification of func GoToTarget

### PR DESCRIPTION
**Pull request by WillardHu**
**What type of PR is this?**

/kind bug

**What this PR does / why we need it**:

Data parameters could not be correctly verified  before repair.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: cloud/pkg/router/provider/eventbus/eventbus.go [hunk 1]
```diff
@@ -112,18 +112,30 @@ func (*EventBus) Forward(target provider.Target, data interface{}) (response int
 
 func (eb *EventBus) GoToTarget(data map[string]interface{}, stop chan struct{}) (interface{}, error) {
 	messageID, ok := data["messageID"].(string)
-	body, ok := data["data"].([]byte)
-	param, ok := data["param"].(string)
+	if !ok {
+		err := errors.New("data transform failed, messageID must be not null")
```
Comment (gy95) [inline cloud/pkg/router/provider/eventbus/eventbus.go:116]: better to print type is not matched, !ok happens when  maybe value is nil or type does not match string

Comment (WillardHu) [reply cloud/pkg/router/provider/eventbus/eventbus.go]: Ok, fixed

**Review by fisherxu — COMMENTED** [review]
/approve

**Comment by WillardHu** [comment]
> WillardHu force-pushed the WillardHu:fix/eventbus branch from b273023 to 19c4e28 40 seconds ago

Batter error message.

**Pull request merged**